.github/workflows/release-binaries.yml

```diff
@@ -2,7 +2,10 @@ name: GoReleaser
 on:
   release:
     types: [ published ]
-
+  workflow_dispatch:
+  workflow_run:
+    workflows: [Releaser]
```

Comment:
Relases created by GitHub actions will not create other workflows, unless they use a GitHub Token other than the built-in default. 

An alternative solution is to delete and recreate the release manually.

I [pointed this out](https://filecoinproject.slack.com/archives/C03KLC57LKB/p1693986844074549) to IPDX team a while back and I believe there has been fixes rolled out to the unified CI? It's worth checking with them.

Reply:
🤷 this seems to work in the lassie repo where we have essentially the same setup and no custom token configured

Reply:
https://github.com/filecoin-project/lassie/actions/runs/9691067118

> Triggered via workflow run 3 months ago

Reply:
Oh nice